back/spec/models/user_spec.rb

```
@@ -42,11 +42,18 @@
 
   describe 'creating a light user - email & locale only' do
     it 'is valid and generates a slug' do
+      SettingsService.new.activate_feature! 'user_confirmation'
       u = described_class.new(email: 'test@test.com', locale: 'en')
       u.save
       expect(u).to be_valid
       expect(u.slug).not_to be_nil
     end
+
+    it 'is still valid if user confirmation is not turned on' do
+      u = described_class.new(email: 'test@test.com', locale: 'en')
+      u.save
```

I would have a preference to use `save!` to avoid silent failures.